src/ensembl/stories/shared-components/expandable-section/ExpandableSection.stories.scss

```diff
@@ -0,0 +1,87 @@
+@import 'src/styles/common';
+
+.collapsedContent{
+    display: grid;
+    padding:6px 20px;
+    grid-template-columns: [title] 120px [primary] 250px [secondary] auto [example-link] 200px;
+    align-items: center;
+
+    .title{
+        grid-column: title;
+        color: $medium-dark-grey;
+    }
+
+    .primary{
+        grid-column: primary;
+
+        .primaryValue{
+            font-size: 22px;
+        }
+
+        .primaryUnit{
+            color: #6f8190;
+            font-size: 13px;
+            padding-left: 5px;
+        }
+    }
+
+    .secondary{
+        grid-column: secondary;
+
+        .secondaryValue{
+            font-size: 22px;
+        }
+
+        .secondaryUnit{
+            color: #6f8190;
+            font-size: 13px;
+            padding-left: 5px;
+        }
+    }
+
+    .exampleLink{
+        cursor: pointer;
+        color: $blue;
+        grid-column:  example-link;
+    }
+}
+
+
+.primaryStats{
+    padding: 13px 20px;
+    display: grid;
+    grid-template-columns: [title] 120px [stats] auto [example-link] 200px;
+    align-items: flex-start;
+
+    .title{
+        grid-column: title;
+        color: $medium-dark-grey;
+    }
+
+    .stats{
+        grid-column: stats;
+    }
+    .exampleLink{
+        cursor: pointer;
+        color: $blue;
+        grid-column:  example-link;
+    }
+}
+
+
+.secondaryStats{
+    padding: 13px 20px;
+    display: grid;
+    grid-template-columns: [title] 120px [stats] auto;
+    align-items: flex-start;
+
+    .title{
+        font-size: 12px ;
+        grid-column: title;
+        color: $medium-dark-grey;
+    }
+
+    .stats{
+        grid-column: stats;
+    }
+}
```

Comment:
again, 4-space indents and no trailing empty line 😞

Reply:
Fixed it